assets/js/navMap.js

```diff
@@ -1631,12 +1631,19 @@ var navMap = (function() {
     },
 
 
-    "filterByTaxon": function(name, preventRefresh, isPhylo) {
+    "filterByTaxon": function(name, preventRefresh) {
       if (!name) {
         var name = $("#taxonInput").val();
       }
 
-      d3.json(paleo_nav.dataUrl + paleo_nav.dataService + '/taxa/list.json?name=' + name + '&show=seq', function(err, data) {
+      var selector;
+      
+      if ( name.match(/^txn:|^var:/) )
```

Comment:
Although some browsers will be okay with it, don't forget curly brackets with `if else`. For example, 
````
if () {

} else {

}
````